05-classification.Rmd

```diff
@@ -663,10 +688,19 @@ scaled_cancer <- bake(uc_recipe, unscaled_cancer)
 scaled_cancer
 ```
 
-Now let's generate the two scatter plots, one for `unscaled_cancer` and one for
-`scaled_cancer`, and show them side-by-side. Each has the same new observation
-annotated with its $K=3$ nearest neighbours:
-
+Figure \@ref(fig:05-scaling-plt) shows the two scatter plots side-by-side&mdash;one for `unscaled_cancer` and one for
```

Comment:
I wonder if at some point in this section we want to explain why it's so damn complicated to scale and centre the data... We have to make a `recipe`, add `step`'s to it, `prep` it and then `bake` it, yikes! Can't we just use `scale`? Of course we can, but then we can't transform the test data set. Problem is the learners don't yet see this problem... But I still think we kind of need to tell them there's a good reason coming up as to why we need to do this in this way. And then in the next chapter, we should make sure to draw attention to this (in particular in `predict` where the `recipe` is automatically applied to new data when `predict` is given a `workflow` object and new data).

Reply:
done